packages/@okta/vuepress-site/docs/reference/api/xaas/index.md

```diff
@@ -0,0 +1,493 @@
+---
+title: Anything-as-a-Source (XaaS)
+category: management
+---
+ 
+# Anything-as-a-Source (XaaS) API
+
+<ApiLifecycle access="ea" />
+
+The Okta Anything-as-a-Source (XaaS) API provides a mechanism to synchronize an HR source (the custom identity source) with Okta user profiles in an org. See [Build an XaaS custom client integration](https://6346d2bcc01af01731982563--reverent-murdock-829d24.netlify.app/docs/guides/anything-as-a-source/) for details.
+
+## Get started
+
+Explore the XaaS API: [![Run in Postman](https://run.pstmn.io/button.svg)](linkhere).
```

Comment:
I surveyed a bunch of the other API docs and most of them have the "Run in Postman" button on the next line, without a period.

Reply:
Thanks! I've reformatted this line to match the other API references.